index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+require_once 'rif-collector-app.php';
+$db = returnDb();
+
+function retrieveAllQuery($db) {
+    $query = $db->prepare("SELECT `rifs`.`make`, `rifs`.`model`, `rifs`.`type`, `colors`.`color` AS 'color', `rifs`.`mags_owned`, `rifs`.`power_source`, `rifs`.`sites_visited`, `rifs`.`purchase_date` FROM `rifs` JOIN `colors` ON `rifs`.`color` = `colors`.`id`;");
+    $result = $query->execute();
```

Comment:
Could potentially do a conditional check on the $result and if successful then return $query->fetchAll(); otherwise provide an error message of some sort.. Just in case it fails to execute for some reason